acceptance/localcluster/localcluster.go

```diff
@@ -24,6 +24,7 @@ import (
 	"io"
 	"io/ioutil"
 	"os"
+	"path"
```

Comment:
Now that we have a legitimate use case for `"path"` I'd say `make check` is mistaken here, but either way works.